devel/ninja/ninja-cmake.mk

```diff
@@ -0,0 +1,13 @@
+include $(INCLUDE_DIR)/cmake.mk
+
+CMAKE_OPTIONS += -DCMAKE_GENERATOR="Ninja"
+
+define Build/Compile/Default
+	mkdir -p $(PKG_BUILD_DIR)/$(CMAKE_BINARY_SUBDIR)
+	cd $(PKG_BUILD_DIR)/$(CMAKE_BINARY_SUBDIR) && $(STAGING_DIR_HOSTPKG)/bin/ninja
+endef
+
+define Build/Install/Default
+	DESTDIR=$(PKG_BUILD_DIR)/ipkg-install \
+	$(STAGING_DIR_HOSTPKG)/bin/ninja install -C $(PKG_BUILD_DIR)/$(CMAKE_BINARY_SUBDIR)
```

Comment:
`$(call Ninja,-C $(PKG_BUILD_DIR)/$(CMAKE_BINARY_SUBDIR),DESTDIR=$(PKG_BUILD_DIR)/ipkg-install)` ?

Reply:
Done.

Reply:
This doesn't seem to work. Specifically DESTDIR doesn't seem to get set to ipkg-install.

Reply:
Fixed. Gave up and copied meson.mk.